### DISCUSSION TITLE
Updates ?

### DISCUSSION THREAD

**Discussion by ChrisWUP** [dschablowsky/FordPassWidget#6] [General]
Hi Damian,

hast Du schon eine Idee ob und wann Du eventuell die Updates einbauen würdest ?
Da Du das viel besser und schneller kannst als ich macht es keinen Sinn wenn ich suche da "rumzubasteln".
Deine App hat einen festen Platz auf meinen iPhone.

Soll ich die App mal meinem Ford Händler zeigen oder willst Du eher weniger Traffic auf Deiner Seite ?

Liebe Grüße, schöne Woche,

Chris

**Comment by dschablowsky** [comment]
Grundsätzlich habe ich alle Daten über das Auto zusammen. 
Mein Problem ist aber: 
Das Design eines Widgets ist dynamisch und man kann meines Wissens nach die Position von z.B. Textfeldern nicht fix coden. Als Beispiel nehme ich den Standort: je nach Länger der Position werden 1-2 Zeilen für den Text benötigt. Wird dies als oberes Element genommen, werden die darunter positionierten Elemente nach unten geschoben und es sieht nicht mehr "schön" aus.
Ich versuche immer wieder die Werte anzupassen, sodass ich dem dynamischen entgegenwirke. Aber bisher habe ich keinen Erfolg.

**Reply by ChrisWUP** [reply]
OK, verstanden. Da kann ich leider so gar nicht sinnvoll helfen. Ich übe mich in Geduld :-)

**Comment by dschablowsky** [comment]
> Soll ich die App mal meinem Ford Händler zeigen oder willst Du eher weniger Traffic auf Deiner Seite ?

Grundsätzlich freue ich mich, wenn das Widget verwendet wird. Da mein Script schon veröffentlich wurde, hat wohl auch der eine oder andere Ford Mitarbeiter es schon gesehen. Daher spricht da auch nichts dagegen ;-)

**Reply by ChrisWUP** [reply]
Dann mache ich das mal :-)

**Comment by dschablowsky** [comment]
Das Update wurde endlich veröffentlich!
In der Version 1.1 gibt es nun mehr Elemente und man kann sich bis zu 6 Stück auswählen ;-)

**Comment by ChrisWUP** [comment]
You my Hero :-)

Danke Dir !

**Comment by ChrisWUP** [comment]
Hi Damian,

habe das Update installiert, bei mir leider 3 kleiner Fehler, die auch die Zeit nicht löst (Update ist seid etwa 1 Stunde aktiv)

a.) 1 Tür wird als geöffnet gezeigt, ist es aber nicht (Die erste Version Deiner app zeigt alle Türen als ZU an)
b.) Der Luftdruck wird nu für die vorderen Reifen angelegt, die 2te Zeile hat "..." Punkte.
c.) Einige Sonderzeichen bei den Überschriften (Die kann ich selbst ändern, oder?)

LG Christian
![IMG_E67D48674BF6-1](https://user-images.githubusercontent.com/76739408/108727861-4a901e80-7529-11eb-9b9e-ce55920131a6.jpeg)

**Reply by dschablowsky** [reply]
Hi Christian,
bei den Problemen bringt leider länger warten nichts.

a) Das ist ein Problem von Ford. Ich habe es eben selbst probiert. Fenster offen gelassen und Auto abgeschlossen -> wurde richtig im Widget angezeigt. Per Funkschlüssel Fenster geschlossen -> Update an Ford wurde nicht geschickt. Die Info "Fenster zu" kommt erst, wenn die Zündung bzw. der Motor gestartet wird. Würde das Auto regelmäßig die Infos senden, wäre die Batterie nach 1-2 Tagen leer.
b) Du hast wahrscheinlich ein kleineres iPhone, oder? Änder die Schriftgröße und dann sollte es passen. Schau mal bei `titleFontSize` und `detailFontSizeMedium`. Darüber ist eine kurze Anleitung mit werten, die du probieren kannst. Falls es nicht passt, einfach die beiden weiter anpassen.
c) Jap, kannst du selbern ändern! Da ist beim kopieren wohl was schief gelaufen.

Viele Grüße
Damian

**Comment by ChrisWUP** [comment]
1000 Dank :-)

a.) OK, verstanden (Komisch ist nur das er die offene Tür auch JETZT noch anzeigt, bin nach Hause gefahren und da war der Motor an und alle Türen zu :-) )
b.) ist ein iPhone XS, ist eigentlich so gross wir die 11 und 12 pro´s
c.) Mache ich, Danke

Dir nochmals DANKE fürs Update :-)

LG C.

**Reply by dschablowsky** [reply]
Das hatte ich leider auch bei den Türen und bei der Info, ob das Auto abgeschlossen ist oder nicht.

Hat es denn mit den anderen Werten geklappt?

**Comment by ChrisWUP** [comment]
Bei den anderen Werten bin ich noch nicht.
aktuell bekomme ich immer eine Fehlermeldung:

**Reply by dschablowsky** [reply]
Welche Fehlermeldung bekommst du?

**Comment by ChrisWUP** [comment]
Konntest Du den Screenshot nicht sehen ?
Hier nochmal beigefügt:




LG Christian

**Reply by dschablowsky** [reply]
leider kann ich keine der screenshots sehen

**Comment by ChrisWUP** [comment]
![IMG_2362](https://user-images.githubusercontent.com/76739408/108834382-67326200-75ce-11eb-9e7f-e470463473a0.jpg)
Jetzt besser ?

**Reply by ChrisWUP** [reply]
In der Ford App selbst sehe ich links vorne aber.
Auch Deine App zeigt immer mal wieder das "richtige" Bild mit dem Luftdruck links vorne. Nur wechselt das eben immer ...

**Reply by dschablowsky** [reply]
Hier wäre es interessant zu wissen was in den Daten steht, wenn das passiert.
Bitte dafür `debug = true` und `clearFileManagerOnNextRun = true` setzen.
So werden die Daten vom Server angezeigt. In der Konsole (innerhalb der Scriptable App) müsste der Text erscheinen mit `{"vehiclestatus":` und da stehen die Werte der Reifen. 
Wie laufen sie, wenn der obere Fehler in der Konsole erscheint?

**Comment by ChrisWUP** [comment]
Kümmere ich mich heute Abend mal drum, habe aktuell den Schreibtisch voll….

LG

**Reply by ChrisWUP** [reply]
HI Damian, 

ich habe das noch nicht geschafft, Ball liegt in meinem Feld .... sorry.
Ich melde mich bei Dir sobald ich das geregelt habe :-)

**Reply by dschablowsky** [reply]
Hi Christian,
kein Problem!

Viele Grüße
